src/test/java/de/retest/recheck/cli/subcommands/DiffIT.java

```diff
@@ -61,4 +68,23 @@ public void diff_should_print_differences() throws Exception {
 
 		assertThat( systemOutRule.getLog() ).contains( expected );
 	}
+
+	@Test
+	public void checkFilterNames_should_check_the_filter_names() throws Exception {
```

Comment:
That's kinda obvious...

You are not really testing anything here since you really re-implemented this method from `checkFilterNames`.